ArduCopter/mode_auto.cpp

```diff
@@ -1796,6 +1796,10 @@ bool Copter::ModeAuto::verify_nav_wp(const AP_Mission::Mission_Command& cmd)
         // play a tone
         AP_Notify::events.waypoint_complete = 1;
         gcs().send_text(MAV_SEVERITY_INFO, "Reached command #%i",cmd.index);
+
+        Vector3f dest = copter.wp_nav->get_wp_destination();
```

Comment:
const  Vector3f dest = copter.wp_nav->get_wp_destination() * 0.01f;
dest.z = dest.z * 1.0f

Reply:
I am sorry, but I do not understand what is the meaning for "dest.z = dest.z * 1.0f"?

Reply:
dest.z = dest.z * -1.0f 
miss the - .
I just don't think we should create two vectors to just pass on update . But that is just nipcking

Reply:
maybe dest.z = -(dest.z) ?